code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -136,6 +136,27 @@
 	taste_description = "rich earthy pungent"
 	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
 
+/datum/reagent/consumable/nutriment/clothiment
+	name = "Clothiment"
+	description = "It's not actually a form of nutriment but it does keep Moth people going for a short while..."
+	nutriment_factor = 30 * REAGENTS_METABOLISM
+	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
+	brute_heal = 0
+	burn_heal = 0
+	///Amount of satiety that will be drained when the clothiment is fully metabolized
+	var/delayed_satiety_drain = 30
+
+/datum/reagent/consumable/nutriment/clothiment/on_mob_life(mob/living/carbon/M, delta_time, times_fired)
+	if(M.satiety < 600)
+		M.adjust_nutrition(15)
+		delayed_satiety_drain += 15
+	. = ..()
+
+/datum/reagent/consumable/nutriment/clothiment/on_mob_delete(mob/living/carbon/M)
+	M.adjust_nutrition(-delayed_satiety_drain)
+	to_chat(M, span_warning("Your stomach suddently feels empty!"))
```

Comment:
I don't think this needs a message, we already have the hunger UI

Reply:
Fair, I will remove it.